clusterloader2/pkg/util/prometheus.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Given that this prometheus stuff is specific to measurement, maybe you could move this file to `pkg/measurement`?

Reply:
Moved to `measurement/util`.